Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -401,7 +401,7 @@ - (void)invalidate
 
 void ScrollerMac::mouseEnteredScrollbar()
 {
-    m_pair.ensureOnMainThreadWithProtectedThis([this] {
```

Comment:
I don't understand why these lambdas were not flagged as being unsafe. This capture of this seems as unsafe as it's elsewhere. Perhaps analysis is blocked by after safety failures?

(I did not attempt to do anything here as this wasn't needed for the three files listed in `Source/WebCore/SaferCPPExpectations/UncountedLambdaCapturesCheckerExpectations`.)

Reply:
@rniwa may know. It might have to do with the `ProtectedThis` naming? Making static analysis recognizes that?